src/SourceBuild/content/build.sh

```diff
@@ -195,19 +196,42 @@ if [[ "$ci" == true ]]; then
   fi
 fi
 
+# Never use the global nuget cache folder
+use_global_nuget_cache=false
+
 . "$scriptroot/eng/common/tools.sh"
 
+project="$scriptroot/build.proj"
+targets="/t:Build"
+
+# This repo uses the VSTest integration instead of the Arcade Test target
+if [[ "$test" == true ]]; then
+  project="$scriptroot/test/tests.proj"
```

Comment:
How does `./build.sh --test` scenario work? It seems that it ends up using `tests.proj`. How does it work if it's not also using `build.proj`?

Reply:
@ViktorHofer - Can you clarify this? Is it bypassing build.proj when using the `--test` option?